packages/pipeline-editor/src/canvas.ts

```diff
@@ -14,12 +14,13 @@
  * limitations under the License.
  */
 
-import { NotebookParser } from '@elyra/services';
+import { ContentParser } from '@elyra/services';
 import { IconUtil, pyIcon, rIcon } from '@elyra/ui-components';
 import { PathExt } from '@jupyterlab/coreutils';
 import { DocumentRegistry } from '@jupyterlab/docregistry';
 import { Contents } from '@jupyterlab/services';
 import { LabIcon, notebookIcon } from '@jupyterlab/ui-components';
+//import { stride } from '@lumino/algorithm';
```

Comment:
is there a reason we've left this commented out instead of deleting it?

Reply:
Nope, I was just wary of completely removing it since it was in the file before despite not being used anywhere that I could see. I just didn't know if there was some context that I was missing about what it's purpose was and that we may need to add it back in. Now that I know, I'll just delete